aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -481,9 +481,18 @@ impl AptosVM {
 
     /// Deserialize a module bundle.
     fn deserialize_module_bundle(&self, modules: &ModuleBundle) -> VMResult<Vec<CompiledModule>> {
+        let max_version = if self
```

Comment:
Can we track all these places in a list somewhere so next time when we do v7 we remember to update this?

Reply:
Yes let’s refactor this piece of code later to make sure we have one single entry.